### ISSUE TITLE
Macatalyst methods not implemented ?

### ISSUE THREAD

**Issue by malikberkane** [Kebechet/Maui.RevenueCat.InAppBilling#48]
I am using this great package to use revenuecat for android and iOS, it works well. I also have a Macatalyst App, but it is not working as it seems is not not implemented, although it should be the same methods as the iOS ones for retrieving universal purchases and offerings

**Comment by Kebechet** [comment]
Hello @malikberkane 
thanks for kind words ❤️
Regarding the Mac support, you say that Mac can run same native code as iOS ? And thus we can reuse the same code for both platforms ? (I am Windows user so my Mac knowledge is quite limited)

**Comment by malikberkane** [comment]
Yes from what I see on the revenue cat documentation, the Purchases iOS SDK is also compatible with macOS, so it should be the same implentation in the macatalyst part of the maui project

https://www.revenuecat.com/docs/getting-started/installation/macos

And thanks again your work was very useful to me !

**Comment by Kebechet** [comment]
Thank you for the link. In following week I will take a look if I will be able to implement that.

**Comment by Kebechet** [comment]
sorry but currently I dont have time to research & implement this. If you would like to see this functionality in the package try to create a PR with the Mac functionality implemented and I will review it
